### PR TITLE
Fix participatory_processes default images problem

### PR DESCRIPTION
**Pull request by NicoleeSo13**
- Fix ActiveStorage problem in participatory processes

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -943,4 +944,4 @@ RUBY VERSION
    ruby 2.7.5p203
 
 BUNDLED WITH
-   2.3.6
+   2.3.5
```
Comment (laurajaime) [inline Gemfile.lock:947]: why?

**Review by laurajaime — COMMENTED** [review]
Good!! only one question

File: Gemfile.lock (hunk 1 above)
Comment (NicoleSo13) [reply Gemfile.lock:947]: It was the bundler job, should I quit this file?

Comment (laurajaime) [reply Gemfile.lock:947]: No, but why the version was downgrade to 2.3.5?

Comment (NicoleSo13) [reply Gemfile.lock]: Fixed

**Review by laurajaime — APPROVED** [review]
LGTM

**Pull request merged**